.travis.yml

```diff
@@ -353,17 +355,27 @@ matrix:
           -DCMAKE_VERBOSE_MAKEFILE=ON
           -DCMAKE_TOOLCHAIN_FILE=${VCPKG_DIR}/scripts/buildsystems/vcpkg.cmake
           -DVCPKG_TARGET_TRIPLET=x64-windows-static"
-      stage: windep-vcpkg
-      name: prereq-vcpkg
+      stage: windep-vcpkg1
+      name: prereq-vcpkg1
       install:
         - choco upgrade cmake.install
         - choco install ninja visualstudio2017-workload-vctools -y
       script:
         - df -h
         - env
         - travis_wait ${MAX_TIME_MIN} bin/sh/install-vcpkg.sh openssl
-        - travis_wait ${MAX_TIME_MIN} bin/sh/install-vcpkg.sh grpc
         - travis_wait ${MAX_TIME_MIN} bin/sh/install-vcpkg.sh libarchive[lz4]
+    - <<: *windows
+      stage: windep-vcpkg2
+      name: prereq-keep-vcpkg2
+      install:
+        - choco upgrade cmake.install
+        - choco install ninja visualstudio2017-workload-vctools -y
```

Comment:
Since this uses the same `install` section, it can be left out, right?

Reply:
Probably. I'll try it.